lib/Buckets.js

```diff
@@ -212,12 +183,15 @@ export default class Buckets {
                         method: 'Buckets.getBucketMetrics',
                         cmd: key,
                     });
-                } else if (Array.isArray(item[1])) {
-                    // convert strings to numbers for comparision
+                } else {
                     const m = getMetricFromKey(key, bucket);
-                    // count is enough as it represents the delta of the
-                    // metric between the two timestamps
-                    bucketRes.operations[`s3:${m}`] += item[1].length || 0;
+                    let count = parseInt(item[1], 10);
+                    count = Number.isNaN(count) ? 0 : count;
+                    if (m === 'incomingBytes' || m === 'outgoingBytes') {
```

Comment:
because that's the key we are using in our responses. The `s3:` prefix was added only for the api operations list (s3:putObject, s3:getObject etc.)